libraries/AP_Scripting/docs/docs.lua

```diff
@@ -597,6 +597,20 @@ function Parameter_ud:init_by_info(key, group_element, type) end
 function Parameter_ud:init(name) end
 
 
+-- desc
+---@class rtc
+rtc = {}
+
+-- desc
+---@return integer|nil
```

Comment:
you can also give descriptions to the returns. 
```suggestion
---@return integer|nil -- year
```
ect